src/screens/SwapScreen/TokenSelect.tsx

```diff
@@ -54,39 +64,43 @@ export const TokenSelect: React.FC<TokenSelectProps> = ({ value, options, onSele
         <MobileOverlay>
           <OptionsContainer ref={selectRef}>
             <Container>
-              {media.mobile && (
-                <MobileHeader>
-                  <Text type={TEXT_TYPES.H} primary>
-                    {selectType}
-                  </Text>
-                  <CloseIcon onClick={() => setSelectMenuVisible(false)} />
-                </MobileHeader>
-              )}
-              {/* TODO add when there will be more tokens */}
-              {/* <SearchInput value={searchTerm} onChange={handleSearchChange} /> */}
-            </Container>
-
-            <OptionsHeader>
-              <Text type={TEXT_TYPES.BODY}>Asset</Text>
-              <Text type={TEXT_TYPES.BODY}>Balance</Text>
-            </OptionsHeader>
-            {options.map((option) => (
-              <Option key={option.key} onClick={() => selectOption(option)}>
-                <OptionRightPart key={option.key}>
-                  <img alt={option.symbol} src={option.img} />
-                  <OptionTitle>
-                    <TokenSymbol color={theme.colors.textPrimary} type={TEXT_TYPES.BODY}>
-                      {option.symbol}
-                    </TokenSymbol>
-                    <Text type={TEXT_TYPES.BODY}>{option.title}</Text>
-                  </OptionTitle>
-                </OptionRightPart>
-
-                <Text color={theme.colors.greenLight} type={TEXT_TYPES.BODY}>
-                  {option.balance ?? "0.00"}
+              <Header>
+                <Text type={TEXT_TYPES.TITLE} primary>
+                  {selectType}
                 </Text>
-              </Option>
-            ))}
+                <CloseIconStyled onClick={() => setSelectMenuVisible(false)} />
+              </Header>
+              <SearchInput placeholder=" " value={searchValue} onChange={handleChangeSearch} />
```

Comment:
```suggestion
              <SearchInput placeholder="" value={searchValue} onChange={handleChangeSearch} />
```

Может убрать пробел?

Reply:
тогда он будет считать что placeholder'а нет и поставит дефолтный, а мне нужен пустой